src/Databases/MySql.php

```diff
@@ -98,19 +121,29 @@ public function dumpToFile(string $dumpFile)
      */
     public function getDumpCommand(string $dumpFile, string $temporaryCredentialsFile): string
     {
-        $command = [
-            "\"{$this->dumpBinaryPath}mysqldump\"",
-            "--defaults-extra-file=\"{$temporaryCredentialsFile}\"",
-            '--skip-comments',
-            $this->useExtendedInserts ? '--extended-insert' : '--skip-extended-insert',
-        ];
+        /* if call is made from MySqlTest */
```

Comment:
In general I really dislike `else` statements. Could you please move the logic to determine the right quote to it's own dedicated method? Use early returns in that method. Method could be called something like `determineQuote` (or something better)